spec/default-directory-provider-spec.coffee

```diff
@@ -1,4 +1,5 @@
 DefaultDirectoryProvider = require '../src/default-directory-provider'
+platform = require './spec-helper-platform'
```

Comment:
Don't use spec-helper-platform: name your test with a `#win32` in it, like `it "normalizes disk drive letter on #win32"`.  This will cause Jasmine to only run that spec on Windows.